src/richie/apps/core/templates/richie/icons.html

```diff
@@ -181,8 +181,20 @@
       </g>
     </symbol>
 
+    <symbol id="icon-groups" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+      <path d="M0 6v-1.3q0-0.9 1-1.5 1-0.6 2.6-0.6 0.2 0 0.4 0 0.2 0 0.4 0.1 0.2 0.1 0.4 0.2-0.1 0.3-0.2 0.6-0.1 0.3-0.2 0.6t-0.1 0.6v1.2H0zM6 6v-1.2q0-0.5 0.3-0.9t0.8-0.3q0.5 0 0.9 0.3t0.3 0.9q0 0.5 0.2 0.9t0.6 0.3v1.2H6zM18 6v-1.2q0-0.3-0.1-0.6t-0.2-0.6q0.2-0.1 0.4-0.2 0.2-0.1 0.4-0.1 1.5 0 2.4 1.1t0.9 2.6v1.3H18zM12 0h6v-0.7q0-0.5-0.4-0.9t-0.9-0.4q-0.5 0-0.9 0.4t-0.4 0.9v0.7h-6zM3.5 0q-1.3 0-2.2 0.9t-0.9 2.2q0 1.3 0.9 2.2t2.2 0.9q1.3 0 2.2-0.9t0.9-2.2q0-1.3-0.9-2.2T3.5 0zM12 12q-1.3 0-2.2-0.9t-0.9-2.2q0-1.3 0.9-2.2t2.2-0.9q1.3 0 2.2 0.9t0.9 2.2q0 1.3-0.9 2.2T12 12z"/>
+    </symbol>
+
+    <symbol id="icon-groups" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+      <path d="M0 4v-1.5q0 -1.286 1.375 -2.097q1.375 -0.811 3.009 -0.811q0.231 0 0.447 0.008q0.346 0 0.667 0.018q0.231 0.008 0.458 0.059q-0.154 0.327 -0.229 0.661t-0.075 0.676v1.5H0zM8 4v-1.5q0 -0.8 0.438 -1.464t1.313 -0.536q0.8 0 1.893 0.268t2.357 0.732q0.987 0.536 1.518 1.232t0.531 1.464v1.5H8zM18 4v-1.5q0 -0.497 -0.875 -0.821q-0.875 -0.324 -1.875 -0.324q-0.563 0 -1.146 0.029q0.429 0.008 0.875 0.029q0.346 0.008 0.688 0.059q-0.154 0.327 -0.229 0.661t-0.075 0.676v1.5H18zM9 3h6v-0.375q0 -0.925 -1.281 -1.464t-2.719 -0.536q-1.975 0 -3.239 0.586t-1.764 1.732v0.057zM3.493 -4.5q-2.286 0 -3.771 -1.771t-1.486 -4.229q0 -2.429 1.771 -4.143t4.229 -1.714q2.286 0 3.771 1.771t1.486 4.229q0 2.429 -1.771 4.143t-4.229 1.714zM19.507 -4.5q-2.286 0 -3.771 -1.771t-1.486 -4.229q0 -2.429 1.771 -4.143t4.229 -1.714q2.286 0 3.771 1.771t1.486 4.229q0 2.429 -1.771 4.143t-4.229 1.714zM9 -1.5q-1.25 0 -2.125 -0.875t-0.875 -2.125q0 -1.25 0.875 -2.125t2.125 -0.875q1.25 0 2.125 0.875t0.875 2.125q0 1.25 -0.875 2.125t-2.125 0.875zM9 -7.5q-1.25 0 -2.125 -0.875t-0.875 -2.125q0 -1.25 0.875 -2.125t2.125 -0.875q1.25 0 2.125 0.875t0.875 2.125q0 1.25 -0.875 2.125t-2.125 0.875z"/>
+    </symbol>
+
```

Comment:
Icon looks broken now.

If you do not have a software to easily scale down our icons you can send me then I will transform them.

<img width="87" alt="image" src="https://github.com/openfun/richie/assets/9265241/b272d4c6-f1fd-45aa-9ee2-cd6d16b5884d">

Reply:
Whoops... Fixed